locale/fi/LC_MESSAGES/django.po

```diff
@@ -2350,30 +2355,45 @@ msgid "Inspect submission"
 msgstr "Tutki palautusta"
 
 #: exercise/templates/exercise/exercise_base.html:97
-#: exercise/templates/exercise/exercise_plain.html:134
+#: exercise/templates/exercise/exercise_plain.html:144
 msgid "View all submissions"
 msgstr "Näytä kaikki palautukset"
 
-#: exercise/templates/exercise/exercise_plain.html:93
-#, python-format
+#: exercise/templates/exercise/exercise_plain.html:92
+#, fuzzy, python-format
```

Comment:
You should not commit "fuzzy" translation changes. The gettext tool got confused about some changes and you need to check what has changed and fix the problems. The "fuzzy" word and the lines starting with `#|` must not be committed. You can fix and remove them by modifying the file in a text editor. The point is that you do not leave the fuzzy parts lying around but instead you fix them.